Firestore/Example/Benchmarks/FSTLevelDBBenchmarkTests.mm

```diff
@@ -53,33 +58,21 @@
   return std::string(documentSize, 'b');
 }
 
-NSString *LevelDBDir() {
-  NSFileManager *files = [NSFileManager defaultManager];
-  NSString *dir =
-      [NSTemporaryDirectory() stringByAppendingPathComponent:@"FSTPersistenceTestHelpers"];
-  if ([files fileExistsAtPath:dir]) {
-    // Delete the directory first to ensure isolation between runs.
-    NSError *error;
-    BOOL success = [files removeItemAtPath:dir error:&error];
-    if (!success) {
-      [NSException raise:NSInternalInconsistencyException
-                  format:@"Failed to clean up leveldb path %@: %@", dir, error];
-    }
-  }
-  return dir;
-}
-
 FSTLevelDB *LevelDBPersistence() {
-  NSString *dir = LevelDBDir();
-  auto remoteSerializer = [[FSTSerializerBeta alloc] initWithDatabaseID:DatabaseId("p", "d")];
-  auto serializer = [[FSTLocalSerializer alloc] initWithRemoteSerializer:remoteSerializer];
-  auto db = [[FSTLevelDB alloc] initWithDirectory:dir serializer:serializer];
-
-  NSError *error;
-  BOOL success = [db start:&error];
-  if (!success) {
+  DatabaseId db_id("p", "d");
+  Path path = util::TempDir().AppendUtf8("FSTLevelDBBenchmarkTests");
```

Comment:
Nit: perhaps move initialization of `path` right after the declaration of `db`? As it stands, it's a little far away from its use.

Reply:
Done.